Add some Get Started structure

tmattio opened the pull request:
- Make the macOS fix a general troubleshooting documentation page
- Add headings to the Editor setup page
- Split the toplevel introduction to a separate Tooling guide

tmattio committed:
Add some Get Started structure

christinerose commented on data/tutorials/getting-started/1_02_troubleshooting.md:
```diff
@@ -1,12 +1,16 @@
+title: Installation Troubleshooting
 description: |
-  This page will walk you through the workaround for ARM64 processors on newer Macs.
+  This page contains instruction to troubleshoot common problems you might stumble accross while installing OCaml.
```
```suggestion
  This page contains instructions to troubleshoot common problems you might stumble across while installing OCaml.
```

christinerose commented on data/tutorials/getting-started/1_02_troubleshooting.md:
```diff
@@ -1,12 +1,16 @@
-# Fix Homebrew Errors on Apple M1
+# Installation Troubleshooting
+
+This page contains instruction to troubleshoot common problems you might stumble accross while installing OCaml.
```
```suggestion
This page contains instructions to troubleshoot common problems you might stumble across while installing OCaml.
```

tmattio committed:
Split introduction to OCaml toplevel

tmattio committed:
Update data/tutorials/getting-started/1_02_troubleshooting.md

Co-authored-by: Christine Rose <christinerose@users.noreply.github.com>

tmattio committed:
Update data/tutorials/getting-started/1_02_troubleshooting.md

Co-authored-by: Christine Rose <christinerose@users.noreply.github.com>

tmattio committed:
Slight text change to make the toplevel documentation independent

tmattio committed:
Introduce the Your First Steps section

tmattio committed:
Uniform title

sabine committed:
your first steps -> first steps, first sentence of expressions section

- to resolve repetition between "Your First Steps" and "Your First OCaml Program"
- first sentence of "Expressions and Definitions" reworded

sabine reviewed (approved):
LGTM

Khady commented:
Also about `v2`, it’s not obvious that it’s about the version of ocaml.org and not about the version of the ocaml language for someone who is starting with ocaml now.

sabine commented:
@Khady Yes, that's why we're currently moving the manual from v2.ocaml.org to ocaml.org

sabine commented:
I expect the move will happen this year.
